packages/frontend/src/pages/QuadletsList.svelte

```diff
@@ -76,7 +76,7 @@ let data: (QuadletInfo & { selected?: boolean })[] = $derived(
     }
 
     if (match && searchTerm.length > 0) {
-      match = quadlet.id.includes(searchTerm);
+      match = quadlet.service?.includes(searchTerm) || false;
```

Comment:
```suggestion
      match = quadlet.service?.includes(searchTerm) ?? false;
```

Linter has a preference for `??` operator